### ISSUE TITLE
[Glimmer] Unwanted line breaks within dynamic element class name

### ISSUE THREAD

**Issue by GavinJoyce** [prettier/prettier#6207]
**Prettier pr-6178**
[Playground link](https://deploy-preview-6178--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEAeAJgSwG4AIwA2AhgM4kC8AOiABZwEES7DCYBmuA5hBOgEYBPOLhgAnAK5wAvlOasO3XoOFsiBEtNkt2XHvyEiJmuTsX6VajTJMK9yw5JnUAfJSi5cACXqM3qAPRY2K5QIAA0IBAADjCY0CTIoESiohAA7gAKyQgJKETYEJjo4SB8okRgANZwMADKURWYUJzIYpIRNDAAtgQA6jSY8CQNYHC1OYM4gwLI4GQlTRqiMBnlnF1EyKrqcBEAViQAHgBC5VU1tURdcAAyTXBblrsgB4e1TZwEcACK4hDwjx2EQaoiWs0+mC611EJSioiaMF6RRgNGQAA4AAzA1IaXrlKKzOFwJbYB4RACOf3gq2iuRApAAtFA4HB0KySqI4JTMJzVkR1pskNsNBENF1MK0jKKPl9fv8HkKnhEYEQ+Ej0CjkAAmZXlTAED4AYQgUMFICg0DJIHEGgAKqrcsLNEA)
```sh
--parser glimmer
```

**Input:**
```hbs
<div class="hello {{if goodbye true}} {{if goodbye false}} {{if goodbye true}} {{if goodbye false}} {{if goodbye true}}">
  Hello
</div>

```

**Output:**
```hbs
<div
  class="hello
    {{if goodbye true}}

    {{if goodbye false}}

    {{if goodbye true}}

    {{if goodbye false}}

    {{if goodbye true}}"
>
  Hello
</div>
```

**Expected behavior:**

```hbs
<div
  class="hello
    {{if goodbye true}}
    {{if goodbye false}}
    {{if goodbye true}}
    {{if goodbye false}}
    {{if goodbye true}}"
>
  Hello
</div>
```

**Comment by dcyriller** [comment]
On master we now have the following output:
```
<div
	class="hello {{if goodbye true}} {{if goodbye false}} {{if goodbye true}} {{
		if goodbye false
	}} {{if goodbye true}}"
>
	Hello
</div>
```

**Comment by thorn0** [comment]
Fixed by #10145
